.storybook/main.js

```diff
@@ -2,17 +2,23 @@ module.exports = {
   "stories": [
     "../src/**/*.stories.mdx",
     "../src/**/*.stories.@(js|jsx|ts|tsx)",
-    "../storybook/**/*.stories.mdx",
-    "../storybook/**/*.stories.@(js|jsx|ts|tsx)"
+    "./stories/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
Added this as a place we can put stories we don't want people to use directly but would like to have in the storybook build.